src/v/kafka/latency_probe.h

```diff
@@ -55,6 +55,16 @@ class latency_probe {
              labels,
              [this] { return _produce_latency.internal_histogram_logform(); })
              .aggregate(aggregate_labels)});
+
+        _metrics.add_group(
+          prometheus_sanitize::metrics_name("fetch_stats"),
+          {sm::make_counter(
+             "poll_count",
+             sm::description("Amount of times we polled partitions for "
```

Comment:
I believe that we use the metric descriptions for chart titles in our auto-generated dashboards. While not optimal you may want a description that works as both a description for the metric and title for a chart of the metric.

Reply:
Yeah fair, I have simplified it and added more detail in a comment.

However, this one on it's own isn't very useful anyway.